src/common/utility.c

```diff
@@ -377,17 +377,11 @@ dt_logo_season_t dt_util_get_logo_season(void)
   return DT_LOGO_SEASON_NONE;
 }
 
-cairo_surface_t *dt_util_get_logo(const float size)
+static cairo_surface_t *dt_util_get_svg_img(gchar *logo, const float size)
```

Comment:
static routine should not start with dt_ but just _ (a coding convention).

Reply:
:+1: